### PR TITLE
Content improvements

### PR DESCRIPTION
**Pull request by mmattel**
Some content improvements and internal ink fixes in prerequisites.

Already on staging.

### CODE REVIEW

File: modules/ROOT/pages/prerequisites/index.adoc [hunk 1]
```diff
@@ -95,7 +95,7 @@ a| RAM requirement for [.underline]#_caching only_#
 
 Using the above table and some sample environments from below, you can easily match the sweet spot and possibly extrapolate it to your environment. Consider that more users will do more requests for content or changes regularly, independent of whether they are accessing oCIS via the web interface, via a mobile client or the desktop synchronization client. If this access is not held in memory, the access needs to go through the backend slowing down the overall response and consumes more CPU accordingly.
 
-Note that the table below shows only one xref:background[space] created and no shares or re-shares which would multiply the needed quantity of meta-data.
+Note that the table below shows only one xref:background[spaces] created and no shares or re-shares which would multiply the needed quantity of meta-data.
```
Comment (EParzefall) [inline modules/ROOT/pages/prerequisites/index.adoc:98]: Why the plural s in spaces when it is only one?

File: modules/ROOT/pages/prerequisites/index.adoc [hunk 2]
```diff
@@ -76,7 +76,7 @@ For a single machine using a local posix storage, the Linux kernel caches meta-d
 
 // fixme: different fs have different meta-data block-sizes or can even chain them
 
-With a meta-data size of 4K each, as is used in the EXT4 filesystem, the following RAM is needed to completely cache all meta-data:
+With a meta-data size of `4K` each block as is used in the EXT4 filesystem (see xref:filesystems-and-shared-storage[Filesystems and Shared Storage] for other values), the following RAM is needed to completely cache all meta-data:
```
Comment (EParzefall) [inline modules/ROOT/pages/prerequisites/index.adoc:79]:
```suggestion
With a meta-data size of `4K` for each block, as is used in the EXT4 filesystem (see xref:filesystems-and-shared-storage[Filesystems and Shared Storage] for other values), the following RAM is needed to completely cache all meta-data:
```

Comment (EParzefall) [reply modules/ROOT/pages/prerequisites/index.adoc:79]: Sentence is confusing. Not totally sure if I guessed right here.

File: modules/ROOT/pages/prerequisites/index.adoc [hunk 3]
```diff
@@ -134,9 +134,9 @@ It is assumed in the examples above, that on avarage, the original change was 4
...
+Each space provides a virtual root and occupies at least one meta-data block at its root. Additionally each space provides the complete meta-data information about the underlying content. As you can see, the quantity of meta-data can grow large.
 
-Whenever a file or folder (node) changes its content or metadata somewhere in the path, the ETag gets recalculated and the change is propagated to the spaces root for identifying a change. To let a client detect changes, the discovery phase starts at the spaces root and checks if the ETag has changed since the last discovery going forward in the tree of changed ETags.
+Whenever a file or folder (node) changes its content or meta-data somewhere in the path, the ETag gets recalculated and the change is propagated to the spaces root for identifying a change. To let a client detect changes, the discovery phase starts at the spaces root and checks if the ETag has changed since the last discovery going forward in the tree of changed ETags.
```
Comment (EParzefall) [inline modules/ROOT/pages/prerequisites/index.adoc:139]: metadata without the dash is actually correct: https://www.merriam-webster.com/dictionary/metadata

File: modules/ROOT/pages/prerequisites/index.adoc [hunk 4]
```diff
@@ -234,16 +234,40 @@ It is strongly recommend to use a reverse proxy like {traefik-url}[Traefik], {ng
 
 == Filesystems and Shared Storage
 
-In addition to well known metadata like _name_, _size_ and _mtime_ (time a file was last modified), oCIS allows users to add arbitrary metadata like _tags_ and _comments_. Therefore oCIS requires and works with POSIX-compliant file systems where this metadata can be mapped to extended attributes. S3 storage is supported too, but requires POSIX-compliant local storage for meta data persistance.
+In addition to well known meta-data like _name_, _size_ and _mtime_ (time a file was last modified), oCIS allows users to add arbitrary meta-data like _tags_ and _comments_. Therefore oCIS requires and works with POSIX-compliant file systems where this meta-data can be mapped to extended attributes. S3 storage is supported too, but requires POSIX-compliant local storage for meta-data persistance.
 
-The currently supported oCIS POSIX-compliant file systems are:
+The currently supported oCIS POSIX-compliant file systems are the following. Note that the default blocksize equals the meta-data block size used at the xref:ram-considerations[RAM Considerations], which is on some filesystems definable and if applicable, is for informational purposes only:
```
Comment (EParzefall) [inline modules/ROOT/pages/prerequisites/index.adoc:239]:
```suggestion
The currently supported oCIS POSIX-compliant file systems are the following. Note that the default block size equals the metadata block size used for the xref:ram-considerations[RAM Considerations], which is on some filesystems definable and if applicable, is for informational purposes only:
```

File: modules/ROOT/pages/prerequisites/index.adoc [hunk 5]
```diff
@@ -234,16 +234,40 @@ It is strongly recommend to use a reverse proxy like {traefik-url}[Traefik], {ng
...
-* {btrfs_url}[BTRFS]
-* {ext4_url}[EXT4]
-* {xfs_url}[XFS]
-* {zfs_url}[ZFS]
+[caption=]
+.Local Filesystems
+[width="60%",cols="30%,70%",options="header"]
+|===
+| Name
+| Default Blocksize
```
Comment (EParzefall) [inline modules/ROOT/pages/prerequisites/index.adoc:246]:
```suggestion
| Default Block Size
```

File: modules/ROOT/pages/prerequisites/index.adoc [hunk 6]
```diff
@@ -234,16 +234,40 @@ It is strongly recommend to use a reverse proxy like {traefik-url}[Traefik], {ng
...
+a| {zfs_url}[ZFS]
+| 128K
+|===
+
+[caption=]
+.Shared Filesystems
+[width="80%",cols="26%,90%",options="header"]
+|===
+| Name
+| Default Blocksize
```
Comment (EParzefall) [inline modules/ROOT/pages/prerequisites/index.adoc:266]:
```suggestion
| Default Block Size
```

File: modules/ROOT/pages/prerequisites/index.adoc [hunk 7]
```diff
@@ -234,16 +234,40 @@ It is strongly recommend to use a reverse proxy like {traefik-url}[Traefik], {ng
...
+
+a| {nfs_url}[NFS]
+| The blocksize depends on the `rsize` parameter in the mount options. Defaults to 4K, usually set to 32K.
```
Comment (EParzefall) [inline modules/ROOT/pages/prerequisites/index.adoc:269]:
```suggestion
| The block size depends on the `rsize` parameter in the mount options. Defaults to 4K, usually set to 32K.
```

File: modules/ROOT/pages/prerequisites/index.adoc [hunk 8]
```diff
@@ -234,16 +234,40 @@ It is strongly recommend to use a reverse proxy like {traefik-url}[Traefik], {ng
...
 
-. Shared Filesystems
-* {cephfs_url}[CephFS]
-* {nfs_url}[NFS]
+a| {cephfs_url}[CephFS]
+| CephFS does not have meta-data by design, You have to use a POSIX compliant filesystem from above to store the meta-data.
```
Comment (EParzefall) [inline modules/ROOT/pages/prerequisites/index.adoc:272]:
```suggestion
| CephFS does not have metadata by design, You have to use a POSIX-compliant filesystem (see list above) to store the metadata.
```

**Review by EParzefall — COMMENTED** [review]
unclear in one place

File: modules/ROOT/pages/prerequisites/index.adoc (hunk 1 above)
Comment (mmattel) [reply modules/ROOT/pages/prerequisites/index.adoc:98]: It is a new term created by oC. Spaces is singular and plural...

File: modules/ROOT/pages/prerequisites/index.adoc (hunk 3 above)
Comment (mmattel) [reply modules/ROOT/pages/prerequisites/index.adoc:139]: ah, ok, will fix 👍

Comment (mmattel) [reply modules/ROOT/pages/prerequisites/index.adoc]: fixed

**Pull request merged**